Automattic/woocommerce-payments
# Add Instant deposit button in Simplify deposit UI.

Fixes https://github.com/Automattic/woocommerce-payments/issues/5889.

#### Changes proposed in this Pull Request

Show `InstantDepositButton` on the new deposit UI. That button is currently used in the current UI.

NOTE:
- In the new UI, the button text is `Deposit available funds`. Currently, the button says `Instant deposit`. I change the button text here, so when the feature flag `_wcpay_feature_simplify_deposits_ui` is off, the instant deposit button in the current (old) UI will also show the new text. I think it's OK instead of overengineer the button to support different text.
- Same with `isSecondary` and `isPrimary` attribute for that button. The current (old) UI uses `isSecondary` but the new UI uses `isPrimary`.

#### Testing instructions

To get instant deposit:
* Make sure to use USD as merchant currency.
* Add debit card as payout method. Read https://woocommerce.com/document/woocommerce-payments/deposits/instant-deposits/#add-debit-card.
* Purchase so you have balance > USD 50 using `4000000000000077` which will bypass pending balance ([ref](https://stripe.com/docs/testing#available-balance)).
* Enable feature flag `update_option( '_wcpay_feature_simplify_deposits_ui', '1' );` via [WP Console](https://wordpress.org/plugins/wp-console/).
* Open Payments > Overview page.
* A `Deposit available funds` button should show under the balance boxes.

-------------------


**Post merge**
